svir/dialogs/upload_dialog.py

```diff
@@ -139,9 +139,13 @@ def _update_layer_style(self):
                 f.write(sld)
             os.system('tidy -xml -i %s' % fname)
         headers = {'content-type': 'application/vnd.ogc.sld+xml'}
+        resp = self.session.post(
```

Comment:
Please add a comment like this: ```
NOTE: to save the style, we actually need to do both the post and put in this sequence, otherwise it doesn't work. We still haven't found any better way to obtain the same correct behavior
```